stream/stream_mf.c

```diff
@@ -37,5 +37,5 @@ const stream_info_t stream_info_mf = {
     .name = "mf",
     .open = mf_stream_open,
     .protocols = (const char*const[]){ "mf", NULL },
-    .stream_origin = STREAM_ORIGIN_FS,
+    .stream_origin = STREAM_ORIGIN_FS | STREAM_ORIGIN_UNSAFE,
```

Comment:
what's the concrete reason for mf to be unsafe?

Reply:
It does more that expected, passing user input to `glob` alone is unsafe and we have more parsing and matching which is not well validated in this demuxer.

Reply:
> passing user input to glob alone is unsafe

is it? why?

> we have more parsing and matching which is not well validated in this demuxer.

we've had a CVE in this demuxer before and it is already only allowed to be referenced from local files